src/services/accounts/index.js

```diff
@@ -65,12 +66,38 @@ export default class AccountsService {
     return true;
   }
 
+  static async validMnemonicWithAccount(accountStore: string, profile: Profile, mne: Mnemonic): Promise<boolean> {
+    try {
+      await panthalassaStop();
+      // eslint-disable-next-line no-empty
+    } catch (e) {
+      // We ignore exception, since we just need stop it in case it was started earlier.
+    }
+    const config = JSON.stringify({
+      encrypted_key_manager: accountStore,
+      // signed_profile: signedProfile,
```

Comment:
Please remove the comment if it's not needed

Reply:
done